packages/x-components/jest.config.js

```diff
@@ -1,5 +1,9 @@
 module.exports = {
-  preset: '@vue/cli-plugin-unit-jest/presets/typescript',
+  preset: 'ts-jest',
+  transform: {
+    '^.+\\.vue$': '@vue/vue2-jest'
+  },
   testMatch: ['<rootDir>/src/**/*.spec.ts'],
-  setupFilesAfterEnv: ['./jest.setup.ts']
+  setupFilesAfterEnv: ['./jest.setup.ts'],
+  testEnvironment: 'jsdom'
```

Comment:
This is because of [this](https://jestjs.io/blog/2021/05/25/jest-27#flipping-defaults)

Reply:
please add a comment with that link, to know in the future